core/src/main/java/org/testcontainers/dockerclient/transport/TestcontainersDockerCmdExecFactory.java

```diff
@@ -0,0 +1,304 @@
+package org.testcontainers.dockerclient.transport;
+
+import com.github.dockerjava.api.command.DockerCmdExecFactory;
+import com.github.dockerjava.core.AbstractDockerCmdExecFactory;
+import com.github.dockerjava.core.DockerClientConfig;
+import com.github.dockerjava.core.SSLConfig;
+import com.github.dockerjava.core.WebTarget;
+import com.github.dockerjava.netty.NettyWebTarget;
+import io.netty.bootstrap.Bootstrap;
+import io.netty.channel.*;
+import io.netty.channel.epoll.EpollDomainSocketChannel;
+import io.netty.channel.epoll.EpollEventLoopGroup;
+import io.netty.channel.kqueue.KQueueDomainSocketChannel;
+import io.netty.channel.kqueue.KQueueEventLoopGroup;
+import io.netty.channel.nio.NioEventLoopGroup;
+import io.netty.channel.socket.DuplexChannel;
+import io.netty.channel.socket.SocketChannel;
+import io.netty.channel.socket.nio.NioSocketChannel;
+import io.netty.channel.unix.DomainSocketAddress;
+import io.netty.channel.unix.UnixChannel;
+import io.netty.handler.codec.http.HttpClientCodec;
+import io.netty.handler.ssl.SslHandler;
+import io.netty.handler.timeout.IdleState;
+import io.netty.handler.timeout.IdleStateEvent;
+import io.netty.handler.timeout.IdleStateHandler;
+import io.netty.util.concurrent.DefaultThreadFactory;
+import org.apache.commons.lang.SystemUtils;
+import org.bouncycastle.jce.provider.BouncyCastleProvider;
+import org.testcontainers.DockerClientFactory;
+
+import javax.net.ssl.SSLEngine;
+import javax.net.ssl.SSLParameters;
+import java.io.IOException;
+import java.net.SocketTimeoutException;
+import java.security.Security;
+import java.util.concurrent.ThreadFactory;
+import java.util.concurrent.TimeUnit;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+/**
+ * This class is a modified version of docker-java's NettyDockerCmdExecFactory v3.1.0-rc-2
+ * Changes:
+ * - daemonized thread factory
+ * - thread group
+ * - the logging handler removed
```

Comment:
Does this mean no more log noise from netty?

If so, 😻 x 💯!
Also, it'd be worthy of a mention in the changelog.

Reply:
:D Yes, it does :) 
It of course makes logs less debuggable but since not every TC user knows how to configure the logger IMO this is a much better UX